### PR TITLE
Npm upgrade

### PR DESCRIPTION
**Pull request by tschallacka**
Hey Kevin,

I hope you don't mind that I wrote your library into an npm module compatible library, that has some self cleanup features and more. I wrote this because I ran into memory leak issues due to the many jQuery instantiations in the original.

Please read examples/page3.npm.html for details for all the added features and such.

I left the original file of liquid slider untouched, but wrote a second library based on the original, and also switched to gulp for compiling the demo that's used in the .npm.html examples, that way I didn't need to touch your grunt files. This way you can also offer legacy support for people that don't wish to rely on node js for their application management.

I haven't made a module that can run without the node environment with in mind that the legacy item is still there for those who want to directly implement it, and anyone serious will put all their files in one index.js that's run through a minifyer.

Here and there I also implemented some alterations, moved some methods around, but by and large it's still your code that you should be able to recognize.

If you could publish this with page 3 keeping the references to me as a small credit that would be much appreciated.

Basically all you have to do now is npm publish.

### CODE REVIEW

**Comment by KevinBatdorf** [comment]
Hi @mdibbets Thank you for this (and sorry for the delay), but I'd rather not merge this in since the plugin is really old now and I don't have a complete understanding of the changes.
